k8s/hammerdb/deployment-mariadb-ephemeral.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mariadb-ephemeral
+  labels:
+    component: mariadb-ephemeral
+spec:
+  selector:
+    matchLabels:
+      component: mariadb-ephemeral
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        component: mariadb-ephemeral
+    spec:
+      containers:
+        - image: mariadb:10.3.34
+          name: mariadb
+          env:
+            - name: MARIADB_ROOT_PASSWORD
+              value: pass
+          ports:
+            - containerPort: 3306
+              name: mariadb
+          volumeMounts:
+            - name: vol-mariadb
+              mountPath: /var/lib/mysql
+            - name: vol-mariadb-conf
+              mountPath: /etc/mysql/my.cnf
+              subPath: my.cnf
+            - name: vol-mariadb-conf
+              mountPath: /etc/mysql/mariadb.cnf
+              subPath: mariadb.cnf
+            - name: vol-mariadb-conf
```

Comment:
Same volume name question as in deployment-hammerdb.yaml

Reply:
That is actually fine since they are in 2 different containers.  Furthermore, they need to be in the same location as this is where mariadb looks for it.